avatars/animationHelpers.js

```diff
@@ -1060,6 +1085,11 @@ export const _applyAnimation = (avatar, now, moveFactors) => {
 
         _handleDefault(spec);
 
+        if (!activeAvatar.unuseAnimation) return;
```

Comment:
It's a bad idea to ever early return in this code. It makes the overall flow of the logic (which is the most important part of this code) impossible to analyze at a glance.

Reply:
Added `isLastBone`, so can make sure `activeAvatar.unuseAnimation` changed in last bone, then no need `return` any more, is it ok? https://github.com/webaverse/app/pull/2791/commits/f5392da41f4c5682b670a371559fe154cfeea226